docs/porting/libraries.md

```diff
@@ -1,5 +1,151 @@
+
+1. Understand where in the code each incompatible type is being used.
+
+   a.  Understand the nature of the incompatible types.  Are they small in number, but used frequently?  Are they large in number, but used infrequently?  Is their use concentrated, or is it spread throughout your code?
+   
+   b. Is it easy to isolate code that's largely incompatible from code that's large compatible?
+   
+   c. Would you need to refactor your code so that you can deal with incompatible types better?
+
+2. If you have assemblies that are mostly incompatible, is it worth leaving them on .NET Framework for now?  Note that this will depend heavily on the architecture of your system.
+
+   a. You may have some functionality in your library that's incompatible with .NET Core because it relies too heavily on .NET Framework or Windows-specific functionality.  Is it worth leaving that functionality behind for now and releasing a .NET Core verson of your library with less features for the time being?
+
+3. Are there alternative .NET Core APIs for incompatible .NET Framework APIs you're using?
+
+   a. For example, if you're using `WebClient` you may be able to use `HttpClient` instead.
+   
+   b. You may also consider using a slightly different .NET Core API to do some work even if it's not a drop-in replacement.  For example, if you don't require XML Schema discovery, you could use Linq to Xml to parse XML data instead of using XMLSchema to handle the parsing.
+   
+4. Is it reasonable to write some code a .NET Framework API was doing yourself?
+
+   a. Consider modifying and using code from the [.NET Framework Reference Source](referencesource.microsoft.com).  It's licensed under the [MIT License](https://github.com/Microsoft/referencesource/blob/master/LICENSE.txt) so you have significant freedom in doing this.
+   
+This analysis phase could take some time.  You may find that you'll have to refactor your code or make significant changes to how you perform certain tasks.  You may even come to the conlcusion that porting to .NET Core isn't feasible at this time.
+
+## Choosing the project system and getting a basis for compilation
+
+Once you have analyzed the ApiPort report and your codebase, the next step is to actually begin porting your code!
+
+### Making a choice: xproj or Portable Class Library?
+
+The first thing you'll need to do is decide on the project system you want to use.  You have two choices: `xproj` or `csproj` via a Portable Class Library (PCL).  Deciding which to pick comes down to one simple question:
+
+Do you have any need to support anything more than just having Visual Studio recognize your project?  Examples of this may be localized resources in a `.resx` folder, working seamlessly with a custom build process, and others.
+
+If yes, you should create a new PCL and select .NET Core as the target.  If you have no needs other than building and running your code in Visual Studio, you can select an xproj project type.
+
```

Comment:
I may be worthwhile also calling which capabilities are currently unique to xproj (producing a NuGet package and cross compilation).